src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -125,4 +133,55 @@ Integer getInt(Map<Option, ?> options) {
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
   // TODO(ajaykannan): implement "Organization" functionality when available (issue #319)
+
+  /** Clears the Policy from a resource. */
+  void clearOrgPolicy(String resource, ClearOrgPolicyRequest request);
+
+  /**
+   * Gets the effective Policy on a resource
+   *
+   * <p>This is the result of merging Policies in the resource hierarchy. The returned Policy will
+   * not have an etag set because it is a computed Policy across multiple resources. Subtrees of
+   * Resource Manager resource hierarchy with 'under:' prefix will not be expanded.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  OrgPolicy getEffectiveOrgPolicy(String resource, GetEffectiveOrgPolicyRequest request);
+
+  /**
+   * Gets the Policy on a resource.
+   *
+   * <p>If no Policy is set on the resource, a Policy is returned with default values including
+   * POLICY_TYPE_NOT_SET for the policy_type oneof. The etag value can be used with
+   * projects.setOrgPolicy() to create or update a Policy during read-modify-write.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  OrgPolicy getOrgPolicy(String resource, GetOrgPolicyRequest request);
+
+  /**
+   * Lists all the Constraints that could be applied on the specified resource.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  Tuple<String, Iterable<Constraint>> listAvailableOrgPolicyConstraints(
```

Comment:
OK, this is new. Definitely change this return type to something less generic.

Reply:
done